src/app/components/Main.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Main = ({ className = '', children }) => {
-    return <main className={`main-area-content bg-white relative flex-item-fluid ${className}`}>{children}</main>;
+const Main = ({ className = '', mainRef, children }) => {
```

Comment:
Rename `mainRef` to `ref` and

```suggestion
const Main = React.forwardRef(({ className = '', children }, ref) => {
```